src/components/LoadFileMenu/index.tsx

```diff
@@ -117,7 +111,9 @@ const LoadFileMenu = ({
             <Dropdown
                 menu={{ items, theme: "dark", className: styles.menu }}
                 placement="bottomRight"
-                disabled={isBuffering || viewerStatus === VIEWER_IMPORTING}
+                disabled={
+                    isBuffering || conversionStatus !== CONVERSION_INACTIVE
+                }
```

Comment:
Should this also be disabled when the status is `SERVER_LIVE`?

Reply:
yes, this should be disabled if we are in any `conersionStatus` other than inactive

Reply:
```suggestion
                disabled={
                    isBuffering || conversionStatus === CONVERSION_ACTIVE
                }
```
Is this what you meant here?

Reply:
![image](https://github.com/simularium/simularium-website/assets/30200665/b8c9879f-c083-4b4d-8d2a-9e3913474b44)

So currently this disables the load button if the user has the page open to the file conversion page, even if no file is selected... This is maybe because I don't have a local Octopus instance running, but the server not communicating probably shouldn't prevent users from being able to access the other import methods?

Reply:
that's how ux wants it